lms/djangoapps/program_enrollments/tests/test_models.py

```diff
@@ -0,0 +1,69 @@
+"""
+Unit tests for ProgramEnrollment models.
+"""
+from __future__ import unicode_literals
+
+from uuid import uuid4
+
+from django.test import TestCase
+
+from lms.djangoapps.program_enrollments.models import ProgramEnrollment, User
+from student.tests.factories import UserFactory
+
+
+class ProgramEnrollmentModelTests(TestCase):
+    """
+    Tests for the ProgramEnrollment model.
+    """
+    def setUp(self):
+        """
+        Set up the test data used in the specific tests
+        """
+        super(ProgramEnrollmentModelTests, self).setUp()
+        UserFactory.create()
+        self.user = User.objects.first()
+        self.email = 'foo@bar.com'
```

Comment:
For each of these values, if you don't need them outside of creating a `ProgramEnrollment`, just use the values directly inside the `create(...)` call.